### DISCUSSION TITLE
No way to add/invite friends on Heroic?

### DISCUSSION THREAD

**Discussion by popcar2** [Heroic-Games-Launcher/HeroicGamesLauncher#1507] [Q&A] [answered]
Is there no way to add friends or invite them to play on Heroic? I was playing Fall Guys and I couldn't find a way to do this. Surprisingly I couldn't find a straight answer to this question anywhere.

**Comment by Etaash-mathamsetty** [comment, marked as answer]
enable the EOS overlay for fall guys (u have to use heroic 2.4.0 beta 2), then press shift-F3 to open the overlay, and you should be good to go

**Comment by matze19999** [comment]
How to do this on steam deck?
